scripts/assemble_xc_framework.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/zsh
+
+cd ../
+./gradlew :app-io-shared:assembleSharedReleaseXCFramework
```

Comment:
Debug alone was not enough, so we also ran the assembleXCFramework in Release.

Reply:
Maybe the iOS app currently doesn't use a debug framework so maybe we can remove it?
It is better to be able to use it to streamline the build/

Reply:
I don't have strong opinion about this but there are fastlane script to do this. Maybe iOS enginers are familiar with that way.

Reply:
Do you have any opinion on this? @RyuNen344

Reply:
@takahirom @RyuNen344 @ry-itto 
Our goal is to facilitate iOS development, so of course if there is a better way, we would prefer it.🙌